docs/java/features/resilience/resilience.md

```diff
@@ -0,0 +1,221 @@
+---
+id: resilience
+title: Resilience Capabilities of the SAP Cloud SDK for Java
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Resilience
+description: This article describes how the SAP Cloud SDK for Java can be used to introduce resilience patterns into an application.
+keywords:
+- sap
+- cloud
+- sdk
+- resilience
+- multi tenancy
+- cloud native
+- timeout
+image:
+---
+
+The SAP Cloud SDK for Java provides abstractions for some frequently used resilience patterns like timeout, retry or circuit breaker.
+Applying such patterns helps making an application more resilient against failures it might encounter.
+
+Also check out our [dedicated tutorial](https://developers.sap.com/tutorials/s4sdk-resilience.html) on the topic!
```

Comment:
It makes sense to put it to the end of this doc to keep the reader's flow.

Reply:
I would rather have it at the top so that in case the reader is looking more for a tutorial it is more prominent and can jump there quickly instead of scrolling through the whole thing. Maybe a slight re-wording would help to better integrate it?

Reply:
Update it.